dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/support/ClusterUtilsTest.java

```diff
@@ -59,6 +59,7 @@ public void testMergeUrl() throws Exception {
                 .addParameter(DEFAULT_KEY_PREFIX + QUEUES_KEY, Integer.MAX_VALUE)
                 .addParameter(DEFAULT_KEY_PREFIX + ALIVE_KEY, Integer.MAX_VALUE)
                 .addParameter(DEFAULT_KEY_PREFIX + THREAD_NAME_KEY, "test")
+                .addParameter("." + Constants.ASYNC_KEY, "test")
```

Comment:
use something like HIDE_KEY_PREFIX instead of "."?

Reply:
what's more,I doubt whether your test case  is correct, should not it be `key + HIDE_KEY_PREFIX  + Constants.ASYNC_KEY` ?

Reply:
> 1. I prefer using  `cleaner` rather than `more clean` in your title;
> 2. The logic of clusterUtils test is not that perfect, I try opposite conditional statements in mergeUrl method, both of them pass the test case. I believe it you can make it stronger.

Thank you for the review.
1.Yes, I make a mistake on title.
2.I just add test logic of the code I added, maybe there are other questions in the unit test code. You can submit a pr